src/modules/portfolio/actions/collect-market-creator-fees.js

```diff
@@ -9,7 +9,8 @@ export const collectMarketCreatorFees = (marketId, callback = logError) => (disp
     augur.api.Cash.getBalance({ _address: marketMailboxAddress }, (err, cashBal) => {
       if (err) return callback(err)
       const cashBalance = speedomatic.bignum(cashBal)
-      augur.rpc.eth.getBalance([marketMailboxAddress, 'latest'], (attoEthBalance) => {
+      augur.rpc.eth.getBalance([marketMailboxAddress, 'latest'], (err, attoEthBalance) => {
+        if (err) return callback(err)
```

Comment:
Shouldn't there also be a check here for:
```
if (attoEthBalance == null) return callback(attoEthBalance)
```
like in src/modules/auth/actions/update-assets.js above?

Reply:
I don't think that `eth.getBalance` can return `null` or `undefined`.  Even for a brand new account it should just return 0.

Reply:
Ok, then the check in `src/modules/auth/actions/update-assets.js` should probably be removed.